.github/workflows/release.yml

```
@@ -17,3 +17,4 @@ jobs:
     uses: semantic-release-action/rust/.github/workflows/release-binary.yml@v5
     with:
       disable-semantic-release-cargo: true
+      toolchain: nightly-2024-05-12
```

@andrew-scott-fischer Ah that's right! I forgot about this value

Can we just use

```suggestion
      toolchain: nightly
```

? Then we don't have to worry about keeping it up to date with any particular nightly build, it'll just use latest